pandas/core/indexers.py

```diff
@@ -65,29 +69,24 @@ def is_empty_indexer(indexer, arr_value: np.ndarray) -> bool:
 def check_setitem_lengths(indexer, value, values) -> None:
     """
     Validate that value and indexer are the same length.
-
```

Comment:
can you revert this line removal. Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. see https://www.python.org/dev/peps/pep-0257/